modules/core/src/main/java/org/apache/synapse/config/xml/EnrichMediatorFactory.java

```diff
@@ -187,4 +207,60 @@ public QName getTagQName() {
         return XML_Q;
     }
 
+    /**
+     *
+     * @param sourceElement
+     * @param targetElement
+     *
+     */
+
+    private void validateTypeCombination(OMElement sourceElement, OMElement targetElement) {
```

Comment:
why not have java docs to explain what this does ? ( why it does it in that way etc)